deployments.go

```diff
@@ -69,6 +69,8 @@ type ListProjectDeploymentsOptions struct {
 	ListOptions
 	OrderBy *string `url:"order_by,omitempty" json:"order_by,omitempty"`
 	Sort    *string `url:"sort,omitempty" json:"sort,omitempty"`
+	UpdatedBefore *string `url:"updated_before,omitempty" json:"updated_before,omitempty"`
+	UpdatedAfter *string `url:"updated_after,omitempty" json:"updated_after,omitempty"`
```

Comment:
Please order these using the ordering used in the docs. So that would mean moving `UpdatedAfter` above `UpdatedBefore`. Also make sure that you run `go fmt` before making a PR so your code is properly formatted.

Last thing is that you know used a `*string` type, but the docs mention these are `datetime` fields. So are you sure `*string` is the right type here?